website/docs/reference/warehouse-profiles/impala-profile.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Apache Impala Profile"
+id: "impala-profile"
+---
+
+## Overview of dbt-impala
+
+**Maintained by:** Cloudera    
+**Author:** Cloudera    
+**Source:** [Github](https://github.com/cloudera/dbt-impala)    
+**dbt Cloud:** Currently un-supported    
+**dbt Slack channel** [Link to channel](https://slack.com)     
+
+
+![dbt-impala stars](https://img.shields.io/github/stars/cloudera/dbt-impala?style=for-the-badge)
+
+## Connection Methods
+
+dbt-impala can connect to Apache Impala and Cloudera Data Platform clusters.
+
+The [Impyla](https://github.com/cloudera/impyla/) library is used to establish connections to Impala.
+
+Two transport mechanisms are supported:
+- binary
+- HTTP(S)
+
+The default mechanism is `binary`. To use HTTP transport, use the boolean option `use_http_transport: [true / false]`.
```

Comment:
food for thought, perhaps make the variable `transport` with values of `https` and `binary` with the latter being the default? Can you explain more about the binary transport mechanism?